Form/FormMapper.php

```diff
@@ -78,6 +79,19 @@ public function add($name, $type = null, array $options = array(), array $fieldD
              $fieldName = str_replace('.', '__', $fieldName);
         }
 
+        $classEntity = $this->admin->getClass();
+        if (!isset($options['required']) && property_exists($classEntity, $name)) {
+            $reflectionProperty = new \ReflectionProperty($classEntity, $name);
+            $annotationReader = new AnnotationReader();
```

Comment:
@core23 do you mean it would not work if doctrine annotation doesn't use in my project, or if SonataAdmin doesn't require it and we would not create an object AnnotationReader?